README.md

```diff
@@ -1,2 +1,58 @@
 # koa-npm-resolution
-Koa middleware that transforms NPM package specifiers to relative paths
+
+This is a middleware for Koa servers that resolves NPM package specifiers in ES modules to relative paths for use on the web.  It is currently only intended for use as middleware in development servers to fascilitate build-free testing/iteration.
+
+## What?
```

Comment:
How about laying out an example like this:

This import uses a *bare module specifier*, which won't currently load natively in browsers (until [import maps](https://www.chromestatus.com/feature/5315286962012160) are available):

```js
import {foo} from 'stuff';
```

`koa-npm-resolution` solves this problem by resolving `stuff` using the same rules as [Node `require()`](https://nodejs.org/api/modules.html#modules_all_together), and transforming the import specifier to a path that can be loaded natively by any browser that [supports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Browser_compatibility) ES modules:

```js
import {foo} from './node_modules/stuff/index.js'
```

Reply:
I completely love this and am stuffing it into the README as is.